src/main/java/uk/gov/justice/laa/crime/cfecrime/utils/RequestHandler.java

```diff
@@ -2,44 +2,98 @@
 
 import lombok.experimental.UtilityClass;
 import lombok.extern.slf4j.Slf4j;
-import uk.gov.justice.laa.crime.cfecrime.api.Result.Outcome;
+import uk.gov.justice.laa.crime.cfecrime.Exceptions.UndefinedOutcomeException;
 import uk.gov.justice.laa.crime.cfecrime.api.*;
```

Comment:
coding standard says to avoid wildcard imports

Reply:
I did change this, I don't know when it got changed back to using *